install/playbooks/roles/borg-backup/files/backup.py

```diff
@@ -733,7 +740,7 @@ def main(args):
         manager.mountRepository()
 
         # Called to just initialise an empty repository
-        if args.action == "init" or not manager.repositoryInitialised():
+        if args.action == "init" and not manager.repositoryInitialised():
```

Comment:
The `manager.initRepository` method checks if the repository is initialized before initialising. It also checks the key import, which is not done here. Would it be sufficient to have the following?
```suggestion
        if args.action == "init"
```

Reply:
Yes, you are right, it is redundant.

But if I left it like that, it is clear, without entering the class code, that the repository should not be initialised twice!

It is safer to check it twice rather than forgetting to check and re-initialising a repository (and consequently, loosing the data)

I will think about it.